example/index.js

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { initialize, getConfig, subscribe, APP_READY } from '@edx/frontend-platform';
 import { AppContext, AppProvider } from '@edx/frontend-platform/react';
-import Header from '@edx/frontend-component-header';
+import Header from '@opencraft/frontend-component-header';
```

Comment:
Since this fork is installed using npm aliasing, does it make sense to replace `@edx` with `@opencraft`?

Reply:
No, I think it should refer itself with the correct name as npm alias will manage references in the repository which uses it. We did the same for [footer](https://github.com/open-craft/frontend-component-footer/commit/b49e7bf686331eccb2a7cdb138b72b219560eff1#diff-25eb80999c2e9f9cd03d36d2baf1364ec623fa5e4aed179df535cf3047dca8f2L7).